deluge/ui/console/__init__.py

```diff
@@ -13,4 +13,4 @@
 
 def start():
 
-    Console().start()
+    return Console().start()
```

Comment:
Was there a reason this didn't get returned before? It was being patched in the tests to return this, but I couldn't figure why we didn't want it to return in the real code.

Reply:
I'm not sure that there was any need to return it so seems fine to include it